docker/build_scripts_pypy/install_pypy.sh

```diff
@@ -7,6 +7,11 @@ function get_shortdir {
     $exe -c 'import sys; print("pypy%d.%d-%d.%d.%d" % (sys.version_info[:2]+sys.pypy_version_info[:3]))'
 }
 
+function get_requirements_txt {
+    local exe=$1
+    $exe -c 'import sys; print("requirements-pre-7.3.0.txt" if sys.pypy_version_info < (7,3,0) else "requirements.txt")'
+}
+
```

Comment:
This is correct since pypy < 7.3 does not properly support pip>20 and wheel>0.34

Reply:
Just for the record: I think I actually tried this installing `pip` 20 and `wheel` 0.34, and it seemed to work? (Except that the old directory symlinks got overwritten, since `pp27-pypy_41` is ambiguous, being the new-style tag for both 7.1.x and 7.2.x.)